src/main/java/duke/Utils.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import java.util.StringJoiner;
+
+public class Utils {
+
+    public static String[] splitBetween(String str, String separator) throws DukeException {
+        StringJoiner start = new StringJoiner(" ");
+        StringJoiner end = new StringJoiner(" ");
+
+        int i = 0;
+        boolean after = false;
```

Comment:
Maybe the boolean name should be changed to isAfter to meet the coding standard.